pkg/tokens/builtin/issuer/issuer.go

```diff
@@ -14,6 +14,9 @@ type DpType = string
 const (
 	DpTypeDataplane = "dataplane"
 	DpTypeIngress   = "ingress"
+
+	DataplaneTokenPrefix        = "dataplane-token"
+	EnvoyAdminClientTokenPrefix = "envoy-admin-client-token"
```

Comment:
put both of those into `signing_key.go. `EnvoyAdminClientTokenPrefix ` has nothing to do with DP Tokens